naughty/debian-testing/1543-apparmor-denial-capnames-perfmon-and-bpf

```diff
@@ -0,0 +1 @@
+*apparmor="DENIED" operation="capable" profile="libvirtd" * comm="rpc-worker" capability=39  capname="bpf"
```

Comment:
The commit message talks about `bpf and perfmon` but here I only see `bpf`.

Reply:
I will reword the commit. (I think adding the second pattern would be unnecessary)